[docs] Added info about using gesture navigation on Android (#15457)
expo/expo: docs/pages/versions/unversioned/sdk/navigation-bar.md

Properties are named after style properties; visibility, position, backgroundColor, borderColor, etc.

> This package does nothing when "Gesture Navigation" is enabled on the Android device. There is currently no native API to detect if "Gesture Navigation" is enabled or not.

Reviewer: ```suggestion
This APIs in this package have no impact when "Gesture Navigation" is enabled on the Android device. There is currently no native Android API to detect if "Gesture Navigation" is enabled or not.
```

> This package does nothing when "Gesture Navigation" is enabled on the Android device. There is currently no native API to detect if "Gesture Navigation" is enabled or not.

Reviewer: ```suggestion
This APIs in this package have no impact when "Gesture Navigation" is enabled on the Android device. There is currently no native Android API to detect if "Gesture Navigation" is enabled or not.
```
